lib/broadside/gli/commands.rb

```diff
@@ -24,7 +24,10 @@ def add_command_flags(cmd)
 
 desc 'Bootstrap your service and task definition from the configured definition.'
 command :bootstrap do |bootstrap|
-  add_tag_flag(bootstrap)
+  bootstrap.desc 'Optionally configured tag - without it you cannot bootstrap a task_definition'
+  bootstrap.arg_name 'TAG'
+  bootstrap.flag 'tag', :optional
```

Comment:
GLI is actually kind of amazing, honestly, though it's a little weird to wrap your head around at first